subsys/caf/modules/Kconfig.ble_state

```diff
@@ -17,6 +17,15 @@ menuconfig CAF_BLE_STATE
 
 if CAF_BLE_STATE
 
+config CAF_BLE_STATE_PM
+	bool "Enable bluetooth LE power manager integration"
+	depends on CAF_BLE_STATE
+	depends on CAF_POWER_MANAGER
+	select CAF_KEEP_ALIVE_EVENTS
+	default y
+	help
+	  This enables small module that blocks power down if we have active BLE connection.
```

Comment:
And what will happen afterwards ?
Will the system then power down when the current active connection terminates, or will the code need to request a new power down later, if this setting was enabled and a power down was requested.

I think this help text could be extended with more info.

Reply:
The problem with putting a comprehensive help everywhere is that we would need to maintain copies of it in multiple places. We have help for power manager in rst.

Reply:
i'm not asking about a ten pages documentation, but I think in general we should be better at providing a bit more info.
I feel just a single line like this here is too cheap.

A lot developers don't turn to official documentation until something is not working (myself included).

Besides, what do you mean by maintaining copies ?

Even a re-write of ths docs / user guide will not need the help text in Kconfig to be changed.
A changed Kconfig help text would most likely mean a new setting.

Of course you are right that some of the info will be present in both places, but a re-write of the docs would not mean a change in the Kconfig help, cause the doc is a more complete description with elaborated use cases, where the Kconfig help text would be very specific to that setting, and direct dependencies.

Not an elaborated use-case.

Reply:
There is an error in the option that comes directly from nrf_desktop. Now keep_alive has more general meaning and I have to reword the documentation string here.

Reply:
"This enables keep alive event. This event is processed by power manager and reschedule power down timer. It works only if power manager is in IDLE state. It cannot wake up the device from any sleep states."

Reply:
Ok - at first look I thought it relates to keep alive event - but I like the documentation above to the keep alive event itself - much better than current one.

Reply:
Ok - I have to admit that this decription is not valid. Basically - power manager can still go to power down mode. But the deepest mode would be limited to SUSPENDED if there is active connection - so no power off mode is allowed. I will rephrase it to:
This enables small module that keeps track on active connections.
If there is any active connection the power down modes would be limited to SUSPEND.
Full POWER OFF mode would be only allowed if we have no active connection.